src/main/java/org/springframework/data/couchbase/core/query/OptionsBuilder.java

```diff
@@ -24,9 +24,15 @@
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 import java.time.Duration;
+import java.util.HashSet;
 import java.util.Map;
 import java.util.Optional;
 
+import com.couchbase.client.core.Core;
```

Comment:
Unused import? (not just a NIT, because Core is potentially on the chopping block)